docker/sirius-workflow/Dockerfile

```diff
@@ -1,4 +1,27 @@
-FROM node:14.13.1-alpine3.12 as asset-env
+FROM golang:1.16-alpine as base
+WORKDIR /app
+
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
+
+RUN apk update \
+    && apk add --no-cache \
+    ca-certificates \
+    git \
+    && update-ca-certificates
+
+### Development with hot reload and debugger
+FROM base AS dev
+WORKDIR /app
+
+RUN go get -u github.com/cosmtrek/air && go install github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
You want to make sure this part of the file is excluded from the Docker build for prod in GitHUb actions. I think Use An LPA have a nice pattern where they make devtools the last layer and then strip it off for anything hitting real infra.